cmd/talosctl/cmd/mgmt/cluster/create/clusterops/configmaker/internal/makers/qemu.go

```diff
@@ -450,12 +450,19 @@ func (m *Qemu) initDisks() error {
 
 	m.ForEachNode(func(i int, node *provision.NodeRequest) {
 		node.Disks = slices.Concat(node.Disks, primaryDisks)
+	})
+
+	if err := m.initExtraDisks(); err != nil {
```

Comment:
and the fifth one @Orzelius 

this one is not trivial - the way config patches are laid down, they are sensitive to the order of the disks